main.go

```diff
@@ -191,7 +191,15 @@ func runMessageQueue(wg *sync.WaitGroup) {
 	mq.SetupMQTT()
 	ctx, cancel := context.WithCancel(context.Background())
 	go mq.Keepalive(ctx)
-	go logic.ManageZombies(ctx)
+	go func() {
```

Comment:
why do we need this go routine.

Reply:
for sending peer update on node deletion by `ManageZombies`. main package for no cyclic dependency